### PR TITLE
Add Troubleshooting Fleet Commander

### PR DESCRIPTION
**Pull request by pbrezina**

### CODE REVIEW

File: src/troubleshooting/fleet-commander.rst [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+.. _Fleet Commander: https://fleet-commander.org
+
+How to get Fleet Commander's logs
+*********************************
+
+To enable verbose Fleet Commander's debug logs please set:
+
+.. code-block:: ini
+    :caption: /etc/xdg/fleet-commander-admin.conf
```
Comment (justin-stephenson) [inline src/troubleshooting/fleet-commander.rst:22]: Does a service need to be restarted for the changes to take effect?

**Comment by justin-stephenson** [comment]
A few minor comments, also I don't think the ToC is necessary on this page.

File: src/troubleshooting/fleet-commander.rst (hunk 1 above)
Comment (pbrezina) [reply src/troubleshooting/fleet-commander.rst:22]: It is dbus activated so I don't think so.

**Comment by pbrezina** [comment]
Done. See the new version.

**Comment by justin-stephenson** [comment]
Ack from me, thanks.

**Pull request merged**